consensus/propagation/reactor.go

```diff
@@ -170,7 +170,7 @@ func (blockProp *Reactor) ReceiveEnvelope(e p2p.Envelope) {
 			blockProp.handleCompactBlock(msg, e.Src.ID(), false)
 			schema.WriteProposal(blockProp.traceClient, msg.Proposal.Height, msg.Proposal.Round, string(e.Src.ID()), schema.Download)
 		case *proptypes.HaveParts:
-			blockProp.handleHaves(e.Src.ID(), msg, false)
+			blockProp.handleHaves(e.Src.ID(), msg)
```

Comment:
weird. I have this changed on the evan/recovery/debug-mamo branch but it didn't get merged in #1695